scripts/release.sh

```diff
@@ -51,28 +51,33 @@ echo Testing Gradle plugin...
 
 echo STAGING SUCCESSFUL!
 
-echo $CI
-if [[ $CI == "true" ]]
-then
-  releaseRepositoryAndPushVersion
-else
-  read -p "Do you want to release and push now? [y/N]" -n 1 -r
-  if [[ $REPLY =~ ^[Yy]$ ]]
-  then
-    releaseRepositoryAndPushVersion
-fi
-fi
-
 releaseRepositoryAndPushVersion()
 {
   echo Releasing the repository...
   ./gradlew releaseRepository
 
-  echo Publishing Gradle Plugin to Gradle Plugin Repository...
+  echo Testing Maven plugin...
```

Comment:
I would opt for a new name here, because that step sound awfully familiar to the "testing Maven plugin" step that comes before. I really had to look, how they are different